packages/core/src/components/Badge/Badge.example.jsx

```diff
@@ -5,12 +5,13 @@ import ReactDOM from 'react-dom';
 ReactDOM.render(
   <Fragment>
     <Badge>Default badge</Badge>
-    <Badge variation="info">Variant badge</Badge>
+    <Badge variation="info">Info badge</Badge>
```

Comment:
Same with this, I would say this example is demonstrating the `variation` prop. Maybe something like "Badge with `variation` prop"?